grakn-graql-shell/src/main/resources/jetty-logging.properties

```diff
@@ -22,6 +22,6 @@
 
 # I'm not kidding, don't get rid of this please
```

Comment:
I'll try killing it, but presumably there's a reason I left this ominous warning here :P